docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -146,6 +146,14 @@ You can copy the preceding code block into a file called `yba-values.yaml` and t
 
 If you are looking for a customization which is not listed, you can view all the supported options and their default values by running the `helm show values yugabytedb/yugaware --version {{<yb-version version="preview" format="short">}}` command and copying the specific section to your own values file.
 
+### Specify custom container registry
+
+Follow to use custom container registry. It can be used with a private container registry.
+
+```yaml
+image:
+  commonRegistry: "gcr.io/mycustomregistry"
+```
```

Comment:
``````suggestion
If you have pushed the container images to a custom registry as mentioned in [Pull and push YugabyteDB Docker images to private container registry](../../prepare-environment/kubernetes/#pull-and-push-yugabytedb-docker-images-to-private-container-registry), set the registry address, as follows:

```yaml
# yba-values.yaml
image:
  commonRegistry: "gcr.io/mycustomregistry"
```

If the registry requires authentication, then create a pull secret and pass the name as follows:

```yaml
# yba-values.yaml
image:
  commonRegistry: "gcr.io/mycustomregistry"
  pullSecret: "mycustomregistry-pull-secret"
```
``````